src/tests/functional/plugin/shared/src/behavior/compiled_model/properties.cpp

```diff
@@ -99,6 +99,11 @@ TEST_P(OVClassCompiledModelPropertiesTests, canCompileModelWithPropertiesAndChec
     }
 }
 
+TEST_P(OVClassAutoCheckProfiling, canCompileModelWithProfilingAndCheckProfiling) {
+    auto compiled_model = core->compile_model(model, target_device, properties);
+    OV_ASSERT_NO_THROW(auto enable_profiling = compiled_model.get_property(ov::enable_profiling));
+}
+
```

Comment:
we should avoid enable Auto specific cases in shared tests
and try leverage existing test or use unit tests to cover

Reply:
please refer to the AUTO test case [here](https://github.com/openvinotoolkit/openvino/blob/f55e967dd5a65d2f7f48f8e1197f20f824b7fe28/src/plugins/auto/tests/functional/shared_tests_instances/behavior/ov_executable_network/properties.cpp#L61).